src/main/resources/application.properties

```diff
@@ -6,3 +6,14 @@ spring.datasource.password=root
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 #spring.jpa.show-sql: true
 
+spring.mail.host = smtp.gmail.com
+spring.mail.port = 587
+spring.mail.properties.mail.smtp.starttls.enable = true
+spring.mail.username = bbmemes.1@gmail.com
```

Comment:
don't leave values specific to you (emails, etc)

Reply:
fix